mitodl/open-edx-plugins
# feat: add rapid response xblock

### What are the relevant tickets?
Closes https://github.com/mitodl/rapid-response-xblock/issues/144

### Description (What does it do?)
- Ports over the https://github.com/mitodl/rapid-response-xblock to open-edx-plugins
- Adds the required pants configurations to support package creation of rapid response

### Screenshots (if appropriate):
<img width="1437" alt="Screenshot 2024-03-13 at 7 16 03 PM" src="https://github.com/mitodl/open-edx-plugins/assets/88967643/747b66b5-f180-4ee6-a617-5424fc163077">
<img width="1437" alt="Screenshot 2024-03-13 at 7 16 54 PM" src="https://github.com/mitodl/open-edx-plugins/assets/88967643/3709ca16-ad06-428d-9c86-a31b751c222a">



### How can this be tested?
- Setup up `open-edx-plugins` and generate package build using `pants package ::`. A package will be generated in the `dist` folder named as `rapid-response-xblock-0.9.1.tar.gz`
- Install rapid response in both LMS and CMS shells using `pip install dist/rapid-response-xblock-0.9.1.tar.gz`.
- Configure rapid response using instructions in the ReadMe file in https://github.com/mitodl/rapid-response-xblock
- Check that the rapid response xblock is functioning properly
